gravitee-apim-portal-webui-next/src/services/current-user.service.ts

```diff
@@ -27,6 +27,7 @@ import { User } from '../entities/user/user';
 })
 export class CurrentUserService {
   public user: WritableSignal<User> = signal({});
+  public isUserAuthenticated: WritableSignal<User> = signal({});
```

Comment:
It would be better to have this signal as a `boolean` and have an effect in place: if the user signal is updated, then isUserAuthenticated is then updated. Ideally, this would replace the below `isAuthenticated` method so that all components are getting updated values if the user becomes authenticated or not.

Reply:
Done